config.json

```diff
@@ -93,12 +93,6 @@
       "blackList": []
     },
     "options": {
-      "limits": {
-        "max": 0,
-        "delayMs": 0,
-        "delayAfter": 0,
-        "windowMs": 60000
-      },
```

Comment:
@karmacoma  Removed because these were not in use after moving to WS

Reply:
Ok yes, this good, thanks 👍